data-store/chapter.md

```diff
@@ -792,5 +767,9 @@ Refactoring as I added features was a pleasure!
 
 ### Summary
 
-(Discussion of the concrete: this data store, data stores in general;
-and of the abstract: immutable data on disk and in memory)
+DBDB is a simple database that makes simple guarantees, and yet
+things still became complicated in a hurry. The foremost technique we used to
+manage this complexity was implementing a ostensibly mutable object with an
+immutable data structure. We encourage you to consider this technique the next
```

Comment:
I lost track of that near the end. Feel free to change it back if you like. There were a bunch of things in my edits where I sort of threw them over the wall and mentally decided to rely on Amy's amazing copyediting to fix :)